src/SFA.DAS.AssessorService.Web/Validators/Standard/OptOutStandardVersionViewModelValidator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Linq;
+using FluentValidation;
+using Microsoft.AspNetCore.Http;
+using SFA.DAS.AssessorService.Application.Api.Client.Clients;
+using SFA.DAS.AssessorService.Web.Helpers;
+using SFA.DAS.AssessorService.Web.ViewModels.Standard;
+
+namespace SFA.DAS.AssessorService.Web.Validators.Standard
+{
+    public class OptOutStandardVersionViewModelValidator : AbstractValidator<OptOutStandardVersionViewModel>
+    {
+        public const string OneApprovedVersionRequiredMessage = "One approved standard version is required";
+
+        public OptOutStandardVersionViewModelValidator(IHttpContextAccessor httpContextAccessor, IStandardVersionClient standardVersionApiClient)
+        {
+            var epaoOrgId = EpaOrgIdFinder.GetFromClaim(httpContextAccessor);
+
+            RuleFor(vm => vm.StandardReference)
+                .NotEmpty()
```

Comment:
The check for empty should not be done as this will produce a friendly error message instead of an exception

Reply:
Removed